shell/src/main/java/org/apache/accumulo/shell/commands/CreateTableCommand.java

```diff
@@ -72,7 +72,7 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
       TableNotFoundException, IOException {
 
     final String tableName = cl.getArgs()[0];
-    NewTableConfiguration ntc = new NewTableConfiguration();
+    final NewTableConfiguration ntc = new NewTableConfiguration();
```

Comment:
If you want, you can also make these kinds of things slightly less redundant:

```suggestion
    final var ntc = new NewTableConfiguration();
```

Reply:
Changed in a13c742ef7